core/domain/storage_model_audit_jobs_test.py

```diff
@@ -59,8 +59,8 @@ def test_all_models_have_audit_jobs(self):
 
         # As models.NAMES is an enum, it cannot be iterated. So we use the
         # __dict__ property which can be iterated.
-        for name in models.NAMES.__dict__:
-            if '__' not in name:
+        for name in models.NAMES:
+            if '__' not in name.value:
```

Comment:
Might not be needed, see comments below for details.

Reply:
done